test/dynamicFormControlGroup.test.ts

```diff
@@ -60,43 +60,36 @@ describe('Dynamic form control group component', function () {
         });
     };
 
-    it("renders label if it exists", () => {
+    it("renders conditional label and its contents", () => {
         const rendered = shallowMount(DynamicFormControlGroup, {
             propsData: {
-                controlGroup: fakeFormGroup
+                controlGroup: fakeFormGroup,
+                readonly: true
             }
         });
-
-        const labelCol = rendered.find("label");
+        const conditionalLabel = rendered.find("conditional-label-stub")
+        expect(conditionalLabel.exists()).toBe(true)
+        expect(conditionalLabel.attributes()).toStrictEqual({"controlgroup": "[object Object]", "readonly": "true"})
```

Comment:
Comparing with `[object Object]` is a bit strange... doesn't really tell you anything about the object. If you're most interested in `readonly` attribute then this would suffice:
```
expect(conditionalLabel.attributes()).toMatchObject({readonly: "true"});
```
Or preferably just:
```
expect(conditionalLabel.props("readonly")).toBe(true)
```